scripts/functions/requirements/arch

```diff
@@ -40,9 +40,23 @@ requirements_arch_define_check_raspberry_pi()
   fi
 }
 
+requirements_arch_define_openssl()
+{
+  case "$1" in
+    (ruby-2.3*|ruby-2.2*|ruby-2.1*|ruby-2.0*|ruby-1.9*)
+      requirements_check openssl-1.0
+      __rvm_update_configure_openssl_dir_option "/usr/lib/openssl-1.0"
```

Comment:
On the arch wiki https://wiki.archlinux.org/index.php/RVM they suggest following solutions:

```
$ rvm pkg install openssl
$ rvm remove 2.3.1
$ rvm install 2.3.1 -C --with-openssl-dir=$HOME/.rvm/usr
```

or 

```
$ sudo pacman -S openssl-1.0 # Probably already installed
$ rvm remove <ruby-version>
$ PKG_CONFIG_PATH=/usr/lib/openssl-1.0/pkgconfig:/usr/lib/pkgconfig rvm install <ruby-version>
```

However if I look at the test from the user it seems that with this implementation --with-openssl-dir is only passed to [configure](https://github.com/rvm/rvm/issues/4006#issuecomment-302737031), but not to [make](https://pastebin.com/raw/uvmQAGCH)

@mpapis any suggestion? Maybe I missed something?